eclaire/base.py

```diff
@@ -1,45 +1,28 @@
-# Copyright 2015 KOGAN.COM PTY LTD
```

Comment:
Why are you removing the copyright? this is a public repo.

Reply:
There's already an apache licence in the base directory. I don't think this is required per file

Reply:
That's my assumption

Reply:
tbh I don't know either

Reply:
> Each original source document (code and documentation, but excluding the LICENSE and NOTICE files) SHOULD include a short license header at the top. If the distribution contains documents not covered by CLA, CCLA or Software Grant (such as third-party libraries) then see the policy guide

http://www.apache.org/dev/apply-license.html

I'll bring it back